[18.949270248] 2020-03-31 08:47:00,345 - INFO - launcher: Starting sanity checks
[18.949270248] 2020-03-31 08:47:00,345 - INFO - launcher: Checking for Python 3.5+
[18.949270248] 2020-03-31 08:47:00,345 - INFO - launcher: Checking console encoding
[18.949270248] 2020-03-31 08:47:00,345 - INFO - launcher: Setting console encoding to UTF-8
[18.949270248] 2020-03-31 08:47:00,345 - INFO - launcher: Ensuring we're in the right environment
[19.946336746] 2020-03-31 08:47:00,346 - INFO - launcher: Adding local bins/ folder to path
[316.255092621] 2020-03-31 08:47:00,643 - INFO - launcher: Required checks passed.
[316.255092621] 2020-03-31 08:47:00,643 - INFO - launcher: Optional checks passed.
[316.255092621] 2020-03-31 08:47:00,643 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[0.7089371681213379] 2020-03-31 08:47:01,035 - WARNING - musicbot.config | In config.py::MainThread(18360), line 121 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[0.7089371681213379] 2020-03-31 08:47:01,035 - INFO - musicbot.config | In config.py::MainThread(18360), line 132 in run_checks: Using i18n: config/i18n/en.json
[0.7099342346191406] 2020-03-31 08:47:01,036 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 225 in _setup_logging: Set logging level to INFO
[0.7109313011169434] 2020-03-31 08:47:01,037 - DEBUG - musicbot.json | In json.py::MainThread(18360), line 8 in __init__: Init JSON obj with config/i18n/en.json
[0.7226555347442627] 2020-03-31 08:47:01,049 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 90 in __init__: Starting MusicBot release-260819-4-gf7d0d74
[0.7236249446868896] 2020-03-31 08:47:01,050 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 96 in __init__: Loaded autoplaylist with 2530 entries
[3.8419649600982666] 2020-03-31 08:47:04,168 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 951 in on_ready: Connection established, ready to go.
[3.8419649600982666] 2020-03-31 08:47:04,168 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 342 in _cache_app_info: Caching app info
[4.0785071849822998] 2020-03-31 08:47:04,405 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 773 in _scheck_ensure_env: Ensuring data folders exist
[4.0795040130615234] 2020-03-31 08:47:04,406 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 783 in _scheck_ensure_env: Deleted old audio cache
[4.0795040130615234] 2020-03-31 08:47:04,406 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 789 in _scheck_server_permissions: Checking server permissions
[4.0795040130615234] 2020-03-31 08:47:04,406 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 793 in _scheck_autoplaylist: Auditing autoplaylist
[4.0795040130615234] 2020-03-31 08:47:04,406 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 797 in _scheck_configs: Validating config
[4.0795040130615234] 2020-03-31 08:47:04,406 - DEBUG - musicbot.config | In config.py::MainThread(18360), line 236 in async_validate: Validating options...
[4.0805041790008545] 2020-03-31 08:47:04,407 - DEBUG - musicbot.config | In config.py::MainThread(18360), line 250 in async_validate: Acquired owner id via API
[4.0805041790008545] 2020-03-31 08:47:04,407 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 800 in _scheck_configs: Validating permissions config
[4.0805041790008545] 2020-03-31 08:47:04,407 - DEBUG - musicbot.permissions | In permissions.py::MainThread(18360), line 94 in async_validate: Validating permissions...
[4.0805041790008545] 2020-03-31 08:47:04,407 - DEBUG - musicbot.permissions | In permissions.py::MainThread(18360), line 98 in async_validate: Fixing automatic owner group
[4.0805041790008545] 2020-03-31 08:47:04,407 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 968 in on_ready: Connected: 689756152304500827/茶々子#0874
[4.0805041790008545] 2020-03-31 08:47:04,407 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 976 in on_ready: Owner:     439790171236597760/otyarr#6996

[4.0814986228942871] 2020-03-31 08:47:04,408 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 979 in on_ready: Guild List:
[4.0814986228942871] 2020-03-31 08:47:04,408 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 983 in on_ready:  - Tea Leaf
[4.0814986228942871] 2020-03-31 08:47:04,408 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 1039 in on_ready: Not bound to any text channels
[4.0814986228942871] 2020-03-31 08:47:04,408 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 1065 in on_ready: Not autojoining any voice channels
[4.0825216770172119] 2020-03-31 08:47:04,409 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 266 in _join_startup_channels: Found resumable voice channel Tea Leaf/雑談【通話】
[4.0825216770172119] 2020-03-31 08:47:04,409 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 281 in _join_startup_channels: Attempting to join Tea Leaf/雑談【通話】
[4.8138778209686279] 2020-03-31 08:47:05,140 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 733 in deserialize_queue: Deserializing queue for 649577356939231252
[4.8912632465362549] 2020-03-31 08:47:05,218 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 425 in get_player: Created player via deserialization for guild 649577356939231252 with 1 entries
[4.8912632465362549] 2020-03-31 08:47:05,218 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 297 in _join_startup_channels: Joined Tea Leaf/雑談【通話】
[4.8912632465362549] 2020-03-31 08:47:05,218 - DEBUG - musicbot.entry | In entry.py::MainThread(18360), line 56 in get_ready_future: Created future for None
[4.8922612667083740] 2020-03-31 08:47:05,219 - INFO - musicbot.entry | In entry.py::MainThread(18360), line 292 in _really_download: Download started: https://youtube.com/watch?v=h04CH9YZcpI
[6.3765647411346436] 2020-03-31 08:47:06,703 - INFO - musicbot.entry | In entry.py::MainThread(18360), line 302 in _really_download: Download complete: https://youtube.com/watch?v=h04CH9YZcpI
[6.3775620460510254] 2020-03-31 08:47:06,704 - FFMPEG - musicbot.player | In player.py::MainThread(18360), line 293 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-h04CH9YZcpI-John_Mellencamp_-_Jack_Diane.m4a
[6.3795816898345947] 2020-03-31 08:47:06,706 - DEBUG - musicbot.player | In player.py::MainThread(18360), line 306 in _play: Playing <musicbot.player.SourcePlaybackCounter object at 0x04F7C9D0> using <discord.voice_client.VoiceClient object at 0x0467F3D0>
[6.3805534839630127] 2020-03-31 08:47:06,707 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 460 in on_player_play: Running on_player_play
[6.3805534839630127] 2020-03-31 08:47:06,707 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 709 in serialize_queue: Serializing queue for 649577356939231252
[6.3805534839630127] 2020-03-31 08:47:06,707 - DEBUG - musicbot.bot | In bot.py::MainThread(18360), line 514 in on_player_play: no channel to put now playing message into
[191.1860198974609375] 2020-03-31 08:50:11,512 - INFO - musicbot.bot | In bot.py::MainThread(18360), line 2706 in on_message: 439790171236597760/otyarr#6996: !!shutdown
[191.6135005950927734] 2020-03-31 08:50:11,940 - DEBUG - musicbot.player | In player.py::Thread-7(8764), line 220 in _playback_finished: Deleting file: audio_cache\youtube-h04CH9YZcpI-John_Mellencamp_-_Jack_Diane.m4a
[191.6144976615905762] 2020-03-31 08:50:11,941 - DEBUG - musicbot.player | In player.py::Thread-7(8764), line 225 in _playback_finished: File deleted: audio_cache\youtube-h04CH9YZcpI-John_Mellencamp_-_Jack_Diane.m4a
[191.812150478] launcher-INFO: All done.
